g2p/mappings/langs/und/und_to_ipa.json

```diff
@@ -1,33 +1,33 @@
 [
-    {"in": "a", "out": "a"},
-    {"in": "b", "out": "b"},
-    {"in": "c", "out": "t͡ʃ"},
-    {"in": "d", "out": "d"},
-    {"in": "e", "out": "e"},
-    {"in": "f", "out": "f"},
-    {"in": "g", "out": "ɡ"},
-    {"in": "h", "out": "h"},
-    {"in": "i", "out": "i"},
-    {"in": "j", "out": "ʒ"},
-    {"in": "k", "out": "k"},
-    {"in": "l", "out": "l"},
-    {"in": "m", "out": "m"},
-    {"in": "n", "out": "n"},
-    {"in": "o", "out": "o"},
-    {"in": "p", "out": "p"},
-    {"in": "q", "out": "q"},
-    {"in": "r", "out": "r"},
-    {"in": "s", "out": "s"},
-    {"in": "t", "out": "t"},
-    {"in": "u", "out": "u"},
-    {"in": "v", "out": "v"},
-    {"in": "w", "out": "w"},
-    {"in": "x", "out": "x"},
-    {"in": "y", "out": "j"},
-    {"in": "z", "out": "z"},
-    {"in": "@", "out": "ə"},
-    {"in": "?", "out": "ʔ"},
-    {"in": "'", "out": "ʔ"},
-    {"in": ",", "out": "ʔ"},
-    {"in": ":", "out": ""}
+    { "in": "a", "out": "a" },
```

Comment:
OK, if your editor insists of these reformattings (spaces inside the curly braces), let's make it our norm and change our rule writer accordingly.
In fact, black would revert this change here, although we don't want to run black on .json files because it add that final list comma and that's a json syntax error. What formatter do you have enabled for json in your IDE? I'd like to disable yours or agree on a shared standard please.

Reply:
ach yes, I have[ prettier ](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)installed

Reply:
separate issue for this now too: https://github.com/roedoejet/g2p/issues/285